ext/phpbb/viglink/language/de/viglink_module_acp.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ *
+ * VigLink extension for the phpBB Forum Software package.
+ *
+ * @copyright (c) 2014 phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ */
+
+/**
+ * DO NOT CHANGE
+ */
+if (!defined('IN_PHPBB'))
+{
+	exit;
+}
+
+if (empty($lang) || !is_array($lang))
+{
+	$lang = array();
+}
+
+// DEVELOPERS PLEASE NOTE
+//
+// All language files should use UTF-8 as their encoding and the files must not contain a BOM.
+//
+// Placeholders can now contain order information, e.g. instead of
+// 'Page %s of %s' you can (and should) write 'Page %1$s of %2$s', this allows
+// translators to re-order the output of data while ensuring it remains correct
+//
+// You do not need this where single placeholders are used, e.g. 'Message %d' is fine
+// equally where a string contains only two placeholders which are used to wrap text
+// in a url you again do not need to specify an order e.g., 'Click %sHERE%s' is fine
+//
+// Some characters you may want to copy&paste:
+// ’ » „ “ …
+//
+
+$lang = array_merge($lang, array(
+	'ACP_VIGLINK'					=> 'VigLink',
+	'ACP_VIGLINK_SETTINGS'			=> 'VigLink-Einstellungen',
+	'ACP_VIGLINK_SETTINGS_EXPLAIN'	=> 'VigLink ist ein externer Dienst, der von Benutzern deines Forums erstellte Links monetarisiert, ohne sich auf das Nutzungserlebnis auszuwirken. Wenn Besucher auf diese Links klicken und auf der Ziel-Website zum Beispiel Waren oder Dienstleistungen einkaufen, dann zahlen einige Händler VigLink eine Provision. Hiervon wird ein Anteil an das phpBB-Projekt weitergegeben. Indem du der Verwendung von VigLink und der Weitergabe der Provision an phpBB.com zustimmst, unterstützt du das Open-Source-Projekt und trägst zu seiner finanziellen Absicherung bei.',
```

Comment:
Next try: VigLink ist ein externer Affiliate-Dienst, der Provisionen aus Links generiert, die Benutzer in deinem Forum erstellen. Dazu werden die Links um Tracking-Informationen von VigLink ergänzt, ohne dass sich für den Benutzer beim Aufruf des Links etwas ändert (VigLink Convert). Wenn Besucher auf der Ziel-Website zum Beispiel Waren oder Dienstleistungen einkaufen, dann zahlen einige Händler VigLink eine Provision. Hiervon wird ein Anteil an das phpBB-Projekt weitergegeben. Indem du der Verwendung von VigLink und der Weitergabe der Provision an phpBB Limited zustimmst, unterstützt du das Open-Source-Projekt und trägst zu seiner finanziellen Absicherung bei.